### PR TITLE
Add `@Transient` & `@Ignore` in Models when needed

### PR DESCRIPTION
**Pull request by KevinBoulongne**
Depends on #232

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt [hunk 1]
```diff
@@ -85,24 +88,30 @@ class Message : RealmObject {
...
     var draftUuid: String? = null
-    @Ignore
+    @Transient
     var fullyDownloaded: Boolean = false
-    @Ignore
+    @Transient
     var hasUnsubscribeLink: Boolean = false
-    @Ignore
+    @Transient
     var parentLink: Thread? = null // TODO
```
Comment (sirambd) [inline app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt:101]: You have to use an inversion relation for that

**Review by sirambd — CHANGES_REQUESTED** [review]
Request changes

File: app/src/main/java/com/infomaniak/mail/data/models/Quotas.kt [hunk 2]
```diff
@@ -21,17 +21,22 @@ import io.realm.kotlin.types.RealmObject
...
 
-    /**
-     * Local
-     */
+    //region Local data (Transient)
+    @Transient
     @PrimaryKey
     var mailboxObjectId: String = ""
+    @Transient
     var maxSize: Long = 20L * 1 shl 30 // TODO: Get this value from API?
```
[the remark this answers is not in the document]
Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/data/models/Quotas.kt:40]: Will be fixed in #244

File: app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt (hunk 1 above)
Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt:101]: Impossible 😭 

https://github.com/realm/realm-kotlin/issues/591

**Review by sirambd — APPROVED** [review]

**Pull request merged**